docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.testimonial.scss

```diff
@@ -1,47 +1,70 @@
 .hb-testimonial {
-  margin-top: hb-calculate-rems(40px);
-
-  .field-hs-row-components .ptype-hs-testimonial &,
-  .field-hs-collection-items .ptype-hs-testimonial & {
-    .hs-paragraph-style--well &,
-    .hs-paragraph-style--well-full-width & {
-      margin-top: 0;
-    }
-  }
-
   &__wrapper {
-    @include hb-quote-wrapper;
-    padding-bottom: hb-calculate-rems(34px);
-    margin: 0;
-
+    padding: hb-calculate-rems(40px) hb-calculate-rems(24px);
+    border-width: hb-calculate-rems(1px);
+    border-style: solid;
+    
     @include grid-media-min('lg') {
-      // override hb-quote-wrapper mixin
-      padding-bottom: hb-calculate-rems(34px);
+      padding: hb-calculate-rems(42px) hb-calculate-rems(45px);
     }
 
-    .counted-items-2 &,
-    .counted-items-3 & {
-      @include grid-media-min('sm') {
-        margin: 0;
+    @include hb-themes(('colorful', 'airy')) {
+      &,
```

Comment:
Using a single `&` can be a bit confusing when the parent block has a lot of rules and other nested selectors. It's better to duplicate the rule to make the code easier to read (it won't affect the resulting CSS). Please limit its use to cases when the parent selector is close and the structure is obvious.